apps/condo/domains/billing/utils/countrySpecificQRCodeParsers/index.js

```diff
@@ -0,0 +1,27 @@
+const { get } = require('lodash')
+
+const { RUSSIA_COUNTRY } = require('@condo/domains/common/constants/countries')
+
+const ruParser = require('./ru.qrCodeParser')
+
+/**
+ * @type {{[countryCode: string]: () => string}}
+ */
+const countrySpecificParsers = {
+    [RUSSIA_COUNTRY]: ruParser,
+}
+
+/**
+ * Returns country specific parser or empty one.
+ * @param {string} country
+ * @return {(qrCode: string) => Object<string, unknown>}
+ */
+const getCountrySpecificQRCodeParser = (country) => {
+    const defaultParser = () => ''
```

Comment:
The default parser differs from the return type, is this ok? Maybe it's better to throw an error if country is not supported?